### PR TITLE
station: init at 1.51.1

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change
This PR adds [station](https://getstation.com/) package.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM
executable works
leaf package

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3493,6 +3493,12 @@
     github = "Lassulus";
     name = "Lassulus";
   };
+  lattfein = {
```
Comment (mmahut) [inline maintainers/maintainer-list.nix]: Do you mind splitting this change into a separate commit please?

Comment (lattfein) [reply maintainers/maintainer-list.nix]: Thanks for the review. I've split the change into two commits.

**Comment by lattfein** [comment]
@mmahut `station` has been updated to `1.51.1` recently. Should I update this PR before merge or open another PR after merge?

**Comment by jonringer** [comment]
i would just update this pr, you can also edit the title

**Comment by lattfein** [comment]
Updated to `1.51.1`.

**Comment by lattfein** [comment]
What is the state of this PR? I've been using `station` at work without any problems. Is there anything I can do?

**Review by jonringer — APPROVED** [review]
re-ran `nix-review`, passes on NixOS
diff LGTM (but not super familiar with appImage packaging)
executable seems to work
leaf packge

```
[10 built, 34 copied (46.9 MiB), 12.5 MiB DL]
https://github.com/NixOS/nixpkgs/pull/69007
1 package were build:
station
```

**Pull request merged**